packages/login-ui/components/ConfirmMnemonic/ConfirmMnemonicContainer.js

```diff
@@ -95,17 +92,23 @@ export default class ConfirmMnemonicContainer extends Component {
     )
   }
 
+  checkIfWordAlreadyUsed = (wordItem) =>
+    this.state.confirmPhrase
+      .findIndex((item) => item.index === wordItem.index) !== -1 // Not found by index equals to "word is not used yet"
```

Comment:
I'd add '{' braces around this function to get a better understanding. Now it's look like part of another method

Reply:
Done